index.d.ts

```diff
@@ -0,0 +1,24 @@
+declare interface Comparator {
+    (objectA, objectB): boolean;
+}
+
+declare interface Pairs {
+    keys: any[];
+    values: any[];
+}
+
+declare type createComparator = (createIsEqual) => Comparator;
+declare type createIsSameValueZero = () => Comparator;
+declare type toPairs = (iterable) => Pairs;
+declare type areIterablesEqual = (objectA: any, objectB: any, comparator: Comparator, shouldCompareKeys: boolean) => boolean;
+
+declare type fe = {
+    createCustomEqual: createComparator;
+    deep: Comparator;
+    shallowEqual: Comparator;
+}
+
+export const createCustomEqual: createComparator;
+export const deepEqual: Comparator;
+export const shallowEqual: Comparator;
+export default fe;
```

Comment:
Forgive the question, I'm not a TS guy ... Does this also apply to `import * as` statements? Because there is no actual default export, only the named exports.